### ISSUE TITLE
HTML

### ISSUE THREAD

**Issue by rimmesbe** [austinholly/tea-cozy#1]
Your HTML looks good, well structured. Continue to expand your usage of semantic tags as there are many more: `<main> <section> <article> <aside> <figure>` and many more.
https://webflow.com/blog/html5-semantic-elements-and-webflow-the-essential-guide

https://github.com/austinholly/tea-cozy/blob/master/index.html

**Comment by austinholly** [comment]
Yeah!! I like this one.  The semantic tags make sense to me, I like things organized and logical in general.  That's why I am going to be very good at coding. I clearly envision and believe my love of learning and dedication are making that statement true, regardless of age.
